frontend/src/util/stores.ts

```diff
@@ -96,6 +97,28 @@ const createParticipantSlice: StateCreator<ParticipantSlice> = (set) => ({
     setParticipantLoading: (participantLoading: boolean) => set(() => ({ participantLoading }))
 });
 
+interface RoundSlice {
+    round: Round;
+    setRound: (round: Round) => void;
+    currentActionIndex: number | null;
+    setCurrentAction: (index: number) => void;
+    currentAction: () => Action | null;
+}
+
+const createRoundSlice: StateCreator<RoundSlice> = (set, get) => ({
+    round: [],
+    setRound: (round: Round) => set(() => ({ round })),
+    currentActionIndex: null,
+    setCurrentAction: (index: number) => set(() => ({ currentActionIndex: index })),
+    currentAction: () => {
+        const index = get().currentActionIndex;
+        if (index === null) {
+            return null;
+        }
+        return get().round[index];
+    },
```

Comment:
This is a computed getter function that we could start using more often to simplify getting data that might need some logic/computation to retrieve.